src/devices/Ina226/samples/README.md

```diff
@@ -0,0 +1,124 @@
+﻿# Ina226 Sample Project
+
+### Summary
+
+This is a sample project which uses the same example code in the binding readme. This readme will be a further explanation of each of the properties, telemetry, and commands in this binding.
```

Comment:
👍 for the extensive documentation

Reply:
I always try to be extensive when possible.